src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestStruct.cs

```diff
@@ -75,7 +79,10 @@ public class SimpleTestStruct : ITestClass
                 @"""MyDecimal"" : 3.3," +
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
                 @"""MyDateTimeOffset"" : ""2019-01-30T12:01:02.0000000+01:00""," +
-                @"""MyEnum"" : 2"; // int by default
+                @"""MyEnum"" : 2,"+ // int by default
```

Comment:
nit: add  a space for consistency

```suggestion
                @"""MyEnum"" : 2," + // int by default
```